hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -121,10 +149,16 @@ public ContainerDiff diff(KeyValueContainerData thisContainer, ContainerProtos.C
   /**
    * Returns the container checksum tree file for the specified container without deserializing it.
    */
+  @VisibleForTesting
   public static File getContainerChecksumFile(ContainerData data) {
     return new File(data.getMetadataPath(), data.getContainerID() + ".tree");
   }
 
+  @VisibleForTesting
+  public static File getTmpContainerChecksumFile(ContainerData data) {
```

Comment:
This signature matches `getContainerChecksumFile` for consistency. Changing them both as this suggests would require the caller to know more internal details like where in the container the file(s) belong and whether the path is relative to the container root or not. Also in practice callers already have `ContainerData` to refer to the container in the first place so this ends up being less verbose because callers do not need to unpack the object themselves.